src/pages/AddItem.jsx

```diff
@@ -1,3 +1,245 @@
+import { useState } from "react";
+import addImg from "../assets/images/add-Image.png";
+import Input from "../components/Input";
+
 export default function AddItem() {
-  return <h1>add item</h1>;
+  const [selectedImage, setSelectedImage] = useState(null);
+  const [showWarning, setShowWarning] = useState(false);
+  const [productName, setProductName] = useState("");
+  const [productDescription, setProductDescription] = useState("");
+  const [productPrice, setProductPrice] = useState("");
+  const [tagInput, setTagInput] = useState("");
+  const [tags, setTags] = useState([]);
+
+  const isFormValid =
+    selectedImage &&
+    productName.trim() &&
+    productDescription.trim() &&
+    productPrice.trim();
+
+  const handleImageClick = (event) => {
+    if (selectedImage) {
+      event.preventDefault();
+      setShowWarning(true);
+    } else {
+      setShowWarning(false);
+    }
+  };
+
+  const onImageChange = (event) => {
+    if (selectedImage) {
+      setShowWarning(true);
+      return;
+    }
+
+    const {
+      target: { files },
+    } = event;

```

Comment:
💬 여담
event 인자를 사용하는 부분이 없어서 처음부터 구조분해할당해서 받아도 될 것 같아요~

```suggestion
  const onImageChange = ({ target }) => {
    if (selectedImage) {
      setShowWarning(true);
      return;
    }

    const file  = target.files?.[0];
```